large-records-benchmarks/large-records-benchmarks.cabal

```diff
@@ -722,6 +730,10 @@ executable bench-superrecord
       Experiment.SR_ParseJSON.Sized.R070
       Experiment.SR_ParseJSON.Sized.R080
 
+  if flag(profile-runtime)
+    ghc-options:
+      -O1
```

Comment:
These options should be in `cabal.project` if needed, and not toggled with flags. Then `cabal check` would be happy.

It's not a big ddeal as `large-records-benchmarks` is not meant to be uploaded, yet it's pity to not `cabal check` the other packages.

TBH, benchmarks probably live in a separate project file.

Reply:
Normally, yes. Here, no. They are part of what this package _does_. By removing them, you effectively broke the benchmarks, they suddenly started measuring something they weren't intended to measure.

Reply:
Having a separate project file for the benchmarks is fine with me.